v1/recommend.yaml

```diff
@@ -56,6 +56,39 @@ paths:
             request:
               uri: '{{options.host}}/{domain}/v1/translation/articles/{from_lang}/{{default(seed_article,"")}}'
       x-monitor: false
+  /recommendation/article/morelike/translation/{seed_article}
```

Comment:
Transferring the question from Gerrit:

"One thing that could be interesting to think about - is whether we want to normalize the titles of the 'seed_article' or whether you want to follow on-wiki redirects here, because we do have special middleware for that purpose, we would just need to enable it." 

The title normalization should not be important here since it's mostly used for cache purging and we do not do any active purges. 

Would following redirects for redirect pages be useful?

Reply:
Missing a colon in the end.

Reply:
Yes, following redirects sounds like a good idea. As for normalizing titles, I need the title as it appears on Wikipedia. For example, I need 'Mount Everest' rather than 'Mount_Everest'.

Reply:
Ok, for redirecting you'd need to enable the 'lib/access_check_filter' with `seed_article` as a title option name. I can follow up with this after we fix the PR

Reply:
OK, made the changes below.